src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -53,24 +48,26 @@ internal static IntPtr AllocateSslContext(long options, SafeX509Handle certHandl
 
                 sslContext.sslPtr = libssl.SSL_new(contextPtr);
 
-                libssl.SSL_CTX_free(contextPtr);     
+                libssl.SSL_CTX_free(contextPtr);
 
                 if (IntPtr.Zero == sslContext.sslPtr)
                 {
                     throw CreateSslException("Failed to create SSSL object from SSL context");
-                }      
+                }
+
+                IntPtr memMethod = libcrypto.BIO_s_mem();
 
-                IntPtr memMethodRead = libcrypto.BIO_s_mem();
-                IntPtr memMethodWrite = libcrypto.BIO_s_mem();
-                if ((IntPtr.Zero == memMethodWrite) || (IntPtr.Zero == memMethodRead))
+                if (IntPtr.Zero == memMethod)
```

Comment:
This is a planned item, being tracked via #3362 . All .resx update will be done for all across.